.github/workflows/package.yaml

```
@@ -54,6 +58,48 @@ jobs:
           role-session-name: arkime-s3
           aws-region: us-east-1
 
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: '^1.19.5'
+          cache: true
+
+      # skip this when running onprem
+      # - name: Ensure k3d is present
+      #   uses: nolar/setup-k3d-k3s@v1
+      #   with:
+      #     skip-creation: true    
+
+      - name: Pull BB minimal package needed for testing
+        run: |
+          aws s3 cp --no-progress "${{ env.BB_MIN_S3_PATH }}/${{ inputs.BB_MIN_PACKAGE }}" "${{ inputs.BB_MIN_PACKAGE }}"
+
+      - name: Set up license file
+        run:
+          echo ${{ secrets.SLACK_WEBHOOK }} >> /tmp/demisto.lic
```

Don't think this was supposed to be SLACK_WEBHOOK, I see a LICENSE that may be correct?

```suggestion
          echo ${{ secrets.LICENSE }} >> /tmp/demisto.lic
```

---

Good catch. Updated.

---

Good catch. Updated